t/substring.neon

```diff
@@ -0,0 +1,6 @@
+VAR s: String := "Hello, World!"
+print(substring(s, 0, 5))
+%= Hello
+
+print(substring(s, 5, 10))
+%! StringIndex
```

Comment:
You can't combine %= and %! like this in a single file. If you want to catch a runtime exception as well as match output from previous expression, you'll have to use `TRY`.